platform/web/doc_classes/EditorExportPlatformWeb.xml

```diff
@@ -47,6 +47,10 @@
 		</member>
 		<member name="variant/extensions_support" type="bool" setter="" getter="">
 		</member>
+		<member name="variant/thread_support" type="bool" setter="" getter="">
+			If enabled, the exported game will support threads. It requires [url=https://web.dev/articles/coop-coep]a "cross-origin isolated" website[/url], which can be difficult to setup and has brings some limitations (i.e. not being able to communicate with third-party websites).
+			If disabled, the exported game will not support threads. As a result, it is more prone to performance and audio issues, but will only require to be run on a HTTPS website.
```

Comment:
I'm not even sure it needs to be on HTTPS if threads are disabled, AFAIK a secure context is also a requirement imposed by SharedArrayBuffer. Worth testing to confirm.

Reply:
Yes, HTTPS is required for audio, at least.

Reply:
See the requirements for AudioWorklet (needs HTTPS).
https://developer.mozilla.org/en-US/docs/Web/API/AudioWorklet